radio/src/bluetooth.h

```diff
@@ -19,11 +19,6 @@
  */
 
 enum BluetoothStates {
-#if defined(PCBX9E) && !defined(USEHORUSBT)
```

Comment:
I would prefer we change macros to BLUETOOTH and BLUETOOTH_XXXX here, either BLUETOOTH_HORUS or BLUETOOTH_X9E (or any else if needed)

Reply:
obviously my remark is valid only if we have many BLUETOOTH protocols supported, if we keep only one, you may forget it